mudslide/even_sampling.py

```diff
@@ -145,6 +146,20 @@ def clone(self, spawn_stack: Optional[Any] = None) -> 'EvenSamplingTrajectory':
         if spawn_stack is None:
             spawn_stack = self.spawn_stack
 
+#
+#You should basically always have to returning a 
+#clone and a model.clone will return a copy which 
+#that also copies the resources it will need 
+#So you don’t want this to do anything special 
+#based on the specific model – you just want this 
+#here should always do a clone (in evensampling trajectory) 
+#and then you should make a clone behave sensibly for a 
+#TMModel and every other model. For all the diabatic model – 
+#you can make a base clone – all that does is the return of the copy of the current data  
+#
+        if isinstance(self.model, TMModel):
+            self.model.clone()
+        print("Srizinevensampling")
```

Comment:
None of this is doing anything yet, so remove.

Reply:
removed.